go/pserver/service.go

```diff
@@ -219,7 +219,7 @@ func (s *Service) GetParam(name string, parameter *Parameter) error {
 }
 
 // pserver save checkpoint
-func (s *Service) doCheckpoint() error {
+func (s *Service) doCheckpoint() (err error) {
```

Comment:
a smart feature of named return value!
seems that I ignored so much return error when saving the checkpoint.